src/components/HyperlinkModal.jsx

```diff
@@ -20,7 +17,7 @@ export default class HyperlinkModal extends Component {
     }
   }
 
-  changeHandler = (event) => {
+  changeHandler(event) {
```

Comment:
We should switch back to arrow function definition because it helps us bind the `changeHandler` to this automatically.

If we wanted to do away with the arrow function definition, we would have to explicitly write `this.handleClick = this.handleClick.bind(this)`

Reply:
fixed with [efcbefc](https://github.com/isomerpages/isomercms-frontend/pull/520/commits/efcbefc66441fffbe4d3e055fae809422aa7a386)